### DISCUSSION TITLE
最新版本的Toolbar使用与文档不符

### DISCUSSION THREAD

**Discussion by coderYyz** [antvis/Graphin#466] [Q&A]
官网链接https://antv.vision/graphin-1.x-site/zh/examples/components/toolbar#Custom

是用的实际版本 "@antv/graphin-components": "^2.4.0",

import { Toolbar } from '@antv/graphin-components';

直接使用

<Graphin data={data}>
    ### <Toolbar direction="vertical"/>
  </Graphin>
    
    然而什么效果都没有,查看源码也没有这个模块，请问怎么解决，什么时候解决？

**Comment by pomelo-nwu** [comment]
@coderYyz 文档是1.x的，master分支是2.x的，可以看tag为1.x的代码，源码在这里：https://github.com/antvis/Graphin/blob/v1.6.4-graphin/packages/graphin-components/src/components/toolbar/index.tsx
